models/src/main/scala/it/pagopa/interop/catalogmanagement/model/persistence/events.scala

```diff
@@ -7,6 +7,7 @@ sealed trait Event extends Persistable
 final case class CatalogItemAdded(catalogItem: CatalogItem)                                        extends Event
 final case class ClonedCatalogItemAdded(catalogItem: CatalogItem)                                  extends Event
 final case class CatalogItemUpdated(catalogItem: CatalogItem)                                      extends Event
+// This event leads just the deletion of the descriptor. The old name has been kept to maintain data retrocompatibility
```

Comment:
does t his seem correct to you?
We should not be able to rename events, right?

If this is the case, I could rename this class and keep the protobuf name, but will lead to misalignments.
What do you think?

Reply:
let's keep retrocompatibility